McPicker/Classes/McPickerBarButtonItem.swift

```diff
@@ -33,7 +33,7 @@ open class McPickerBarButtonItem: UIBarButtonItem {
      
         - returns: McPickerBarButtonItem
      */
-    public class func done(mcPicker: McPicker, title: String? = nil, barButtonSystemItem: UIBarButtonSystemItem = .done) -> McPickerBarButtonItem {
+    public class func done(mcPicker: McPicker, title: String? = nil, barButtonSystemItem: UIBarButtonItem.SystemItem = .done) -> McPickerBarButtonItem {
```

Comment:
Does `UIBarButtonItem.SystemItem` work in swift 4.1 too?

Reply:
I tested the build on XCode 10 with Swift 4 language, it built correctly. Testing it on old previous XCode with Swift 4.1 failed on `UIBarButtonItem.SystemItem`

Reply:
If we make it non-reverse compatible we won't have to worry about this